app/scripts/initSentry.js

```diff
@@ -0,0 +1,7 @@
+import setupSentry from './lib/setupSentry'
+
+// setup sentry error reporting
+global.sentry = setupSentry({
+  release: process.env.METAMASK_VERSION,
```

Comment:
I'm guessing that `METAMASK_VERSION` was added so it could be used here?

We use `platform.getVersion()` elsewhere, which calls `extension.runtime.getManifest().version`. This should work here as well I think, since this API is supported in both the background and the contenscript.

Reply:
correct. we wanted to avoid importing anything e.g. the extensionizer package during this limited setup phase